ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -6498,6 +6498,16 @@ static int listen_handler(ldmsd_req_ctxt_t reqc)
 	if (!listen) {
 		if (errno == EEXIST)
 			goto eexist;
+		else if (errno == ENOENT) {
+			reqc->errcode = ENOENT;
+			(void) snprintf(reqc->line_buf, reqc->line_len,
+					"Listen error: authentication domain "
+					"'%s' not found. Please make sure "
+					"that it is created with `auth_add` "
```

Comment:
If -a <auth-plugin> is used on the command line is 'auth_add' still required?

Reply:
If `-a munge` was supplied at the CLI, then `listen` and `prdcr_add` will use it by default if `auth` attribute is not given. However, if `auth=munge` is given, it will look for auth config object named "munge". In this case, `auth_add name=munge plugin=munge` is still needed.